shinydashboard/lantern/modules/endpointsmodule.R

```diff
@@ -117,7 +117,7 @@ endpointsmodule <- function(
   # Create the format for the csv
   csv_format <- reactive({
     res <- selected_fhir_endpoints() %>%
-      select(-updated, -label, -status, -availability) %>%
+      select(-label, -status, -availability) %>%
```

Comment:
Another bug that I just noticed is that the Field Descriptions CSV now doesn't align with the table, since the mime types is now supported formats and the valid capability statement field was added. Could you update that CSV?